src/config/express/express.config.ts

```diff
@@ -46,9 +51,18 @@ export class ExpressConfig {
      * Initializes the route '/' with a welcome message
      */
     private static initServerHome() {
-        this.app.get("/", (req: exp.Request, res: exp.Response) => {
+        this.app.get("/", async (req: exp.Request, res: exp.Response) => {
+            const token = await JwtDriver.generateBearerToken({
+                userId: "123456",
+                username: "john_doe",
+                name: "John Doe",
+                email: "john.doe@example.com",
+                organization: "Example Organization",
+                emailVerified: true,
+                accessGroups: ["group1", "group2"],
+            });
```

Comment:
Why is this defined?

Reply:
Lol this was for me testing to make sure the auth was workign, good catch.